apps/functions-subscription/src/adapters/azure/functions/codec.ts

```diff
@@ -21,3 +23,9 @@ export const toActivationJobAPI = (
   usersActivated: activationJob.usersActivated,
   usersToActivate: activationJob.usersToActivate,
 });
+
+export const toTrialAPI = (trial: Trial): TrialAPI => ({
+  trialId: trial.id,
```

Comment:
**thought**: I'd rename this (and on the OpenAPI as well) to `id`.

what do you think @datalek?

Reply:
Yes, I think `id` is a better name too 👍